### PR TITLE
(#7) Enabler: Set up app scaffolding

### LINKED ISSUE #7: Enabler: Set up app scaffolding
# Set up app scaffolding

Includes:
* Analytics
* Build for React App Dev
* Cypress (with no tests)
* etc.
(Copy/Paste Glossary)

> **ESTIMATE** 5 / 8

### Resources:
* Feature: JIRA Ticket #
* Mockups: *Here goes a URL to or the name of the mockup(s) in inVision*
* References: *URL to web site with examples*
* Web Page: *Here goes a URL to an existing web page*

### Prerequisites
- *required dependency for identifying the problem, scope and then solution for this issue*
- [ ] Short Spike Description - Issue #9999

### Sub-Tasks
- [ ] Short Task Description - Issue #9999
- [ ] Short Task Description - Issue #9999

### Notes

*Some complementary notes if necessary:*

* > Here goes a quote from an email
* Here goes whatever useful information can exist…
* More notes, or technical nuances/direction

### PR DESCRIPTION
**Pull request by seyilonge-nci**
Closes #7 .

https://react-app-dev.cancer.gov/clinical-trials-listing-app/pr-24

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+	let dynamicRoutes;
+	const {
+		CodeOrPurlPath,
+		CodeOrPurlWithTypePath,
+		PurlPath
+	} = useAppPaths();
+	const [{ trialListingPageType }] = useStateValue();
+
+	switch (trialListingPageType) {
+		case 'Disease':
```
[the remark this answers is not in the document]
Comment (seyilonge-nci) [reply src/App.js]: Removed all references and updated comments for #6

File: public/index.html [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+			const configIndex = getUrlParam('cfg', '0');
+			window.currentConfig = configIndex;
+			const cfgs = [
+				{
+					analyticsChannel: 'Clinical Trials',
+					analyticsHandler: 'EddlAnalyticsHandler',
+					analyticsPublishedDate: 'unknown',
+					analyticsContentGroup: 'Clinical Trials: Custom',
+					apiBaseEndpoint: 'https://webapis.cancer.gov/',
+					appId: '@@/DEFAULT_SWS_APP_ID',
```
Comment (arcepaul) [inline public/index.html:67]: update

Comment (seyilonge-nci) [reply public/index.html]: Missed, thanks. Updated.

File: src/App.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (bryanpizzillo) [reply src/App.js]: ¡Muchas gracias, mi amigo!

**Review by arcepaul — APPROVED** [review]

**Pull request merged**